intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -407,54 +414,54 @@ We can additionally explore _overall properties_ of the data frame with two diff
 
 This provides summary statistics for each column:
 
-```{r iris-summary}
-summary(iris)
+```{r penguins-summary}
+summary(penguins)
 ```
 
 This provides a short view of the **str**ucture and contents of the data frame.
 
-```{r iris-str}
-str(iris)
+```{r penguins-str}
+str(penguins)
 ```
 
-You'll notice that the column `Species` is a _factor_: This is a special type of character variable that represents distinct categories known as "levels". 
-We have learned here that there are three levels in the `Species` column: setosa, versicolor, and virginica. 
+You'll notice that the column `species` is a _factor_: This is a special type of character variable that represents distinct categories known as "levels". 
+We have learned here that there are three levels in the `species` column: Adelie, Chinstrap, and Gentoo.
 We might want to explore individual columns of the data frame more in-depth. 
 We can examine individual columns using the dollar sign `$` to select one by name:
 
-```{r iris-subset}
-# Extract Sepal.Length as a vector
-iris$Sepal.Length
+```{r penguins-subset}
+# Extract bill_length_mm as a vector
+penguins$bill_length_mm
 
 # indexing operators can be used too
-iris$Sepal.Width[1:10]
+penguins$bill_depth_mm[1:10]
 ```
 
 We can perform our regular vector operations on columns directly.
 
-```{r iris-col-mean, live = TRUE}
-# calculate the mean of the Sepal.Length column
-mean(iris$Sepal.Length)
+```{r penguins-col-mean, live = TRUE}
+# calculate the mean of the bill_length_mm column
+mean(penguins$bill_length_mm)
```

Comment:
Even though this code is being removed, we should have the final 'correct' answer in the rendered version.
```suggestion
mean(penguins$bill_length_mm,
     na.rm = TRUE) # remove missing values before calculating the mean
```